csync/src/vio/csync_vio_local_win.c

```diff
@@ -230,8 +230,11 @@ int csync_vio_local_stat(const char *uri, csync_vio_file_stat_t *buf) {
     ULARGE_INTEGER FileIndex;
     mbchar_t *wuri = c_utf8_path_to_locale( uri );
 
-    h = CreateFileW( wuri, 0, FILE_SHARE_READ, NULL, OPEN_EXISTING,
-                     FILE_ATTRIBUTE_NORMAL+FILE_FLAG_BACKUP_SEMANTICS+FILE_FLAG_OPEN_REPARSE_POINT, NULL );
+    h = CreateFileW( wuri, 0, FILE_SHARE_WRITE | FILE_SHARE_READ | FILE_SHARE_DELETE,
+                     NULL, OPEN_EXISTING,
+                     FILE_ATTRIBUTE_NORMAL+FILE_FLAG_BACKUP_SEMANTICS+
+                     FILE_FLAG_OPEN_REPARSE_POINT+FILE_FLAG_OVERLAPPED,
+                     NULL );
     if( h == INVALID_HANDLE_VALUE ) {
```

Comment:
I don't think FILE_FLAG_OVERLAPPED is needed, afaik it's only relevant for the opener (=us)..